src/Avatar/__tests__/__snapshots__/Avatar.test.tsx.snap

```diff
@@ -166,7 +166,6 @@ exports[`Avatar Component should render without issues 1`] = `
           },
         }
       }
-      transition={true}
```

Comment:
Why there is a change in the behaviour of the Avatar?
I.e. why default transition is turned off?

Reply:
React Native Image do no have `transition` prop.